### PR TITLE
cpplint recipe

### PR DESCRIPTION
**Pull request by tovrstra**
This recipe builds a package for the popular community maintained fork of Google's cpplint.py script.

### CODE REVIEW

**Comment by marcelotrevisani** [comment]
That is ok, we have some problems with the azure on windows.
I would recommend you to change this recipe to ``noarch: python``

File: recipes/cpplint/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+    url: https://github.com/cpplint/{{ name }}/archive/{{ version }}.tar.gz
+    sha256: {{ sha256 }}
+
+build:
+    number: 0
+    script: "{{ PYTHON }} -m pip install . --no-deps -vv"
```
Comment (marcelotrevisani) [inline recipes/cpplint/meta.yaml]: Please add ``noarch: python``

Comment (tovrstra) [reply recipes/cpplint/meta.yaml]: Good, I'll try. That gave me errors previously with cpplint.exe being a binary (which would surprise me). Let's see.

Comment (marcelotrevisani) [reply recipes/cpplint/meta.yaml]: if you have a binary which needs to be executed in a specific platform, it cannot be ``noarch``
I thought that was a pure python project

File: recipes/cpplint/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+
+source:
+    url: https://github.com/cpplint/{{ name }}/archive/{{ version }}.tar.gz
+    sha256: {{ sha256 }}
+
+build:
+    number: 0
+    noarch: python
+    script: "{{ PYTHON }} -m pip install . --no-deps -vv"
+
```
Comment (chrisburr) [inline recipes/cpplint/meta.yaml]: You need to add `entry_points` for this to work as a `noarch` package:
* https://docs.conda.io/projects/conda-build/en/latest/source/resources/define-metadata.html?highlight=selectors#python-entry-points
* https://github.com/cpplint/cpplint/blob/master/setup.py#L62-L66

Comment (tovrstra) [reply recipes/cpplint/meta.yaml]: Thanks. It works now.

File: recipes/cpplint/meta.yaml (hunk 1 above)
Comment (tovrstra) [reply recipes/cpplint/meta.yaml]: Thanks. it works now.

**Comment by marcelotrevisani** [comment]
LGTM!

**Pull request merged**